packages/eslint-plugin/tests/rules/no-base-to-string.test.ts

```diff
@@ -60,12 +65,27 @@ ruleTester.run('no-base-to-string', rule, {
       `,
     ),
 
+    // variable toLocalString() and template
+    ...literalList.map(
```

Comment:
Oh, let's not add more autogenerated tests. We don't need to re-test so many existing cases for `toLocaleString()`. Let's just stick with a few choice ones that you have below.

Reply:
I removed it in [31a7cf4](https://github.com/typescript-eslint/typescript-eslint/pull/10138/commits/31a7cf4d164836c8613eb10f2db06dba15f9d16e).